### PR TITLE
(fix) (svelte2tsx) also transform element true attribute to lowercase

### PR DESCRIPTION
**Pull request by jasonlyu123**
ex: `<input READONLY>`

### CODE REVIEW

File: packages/svelte2tsx/src/htmlxtojsx/index.ts [hunk 1]
```diff
@@ -384,7 +384,13 @@ export function convertHtmlxToJsx(
         }
 
         //we are a bare attribute
-        if (attr.value === true) return;
+        if (attr.value === true) {
+            if (parent.type === 'Element' && !sapperNoScroll && parent.name !== '!DOCTYPE') {
+                console.log(parent);
```
Comment (dummdidumm) [inline packages/svelte2tsx/src/htmlxtojsx/index.ts:389]: A wild `console.log` appears!

Comment (dummdidumm) [reply packages/svelte2tsx/src/htmlxtojsx/index.ts]: ... okay not anymore 😄

Comment (jasonlyu123) [reply packages/svelte2tsx/src/htmlxtojsx/index.ts]: Fail to remove it before you see it lol

**Review by dummdidumm — APPROVED** [review]
👍

**Pull request merged**